<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notice</class>
 <widget class="QDialog" name="notice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thông báo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/images/notice.png</normaloff>:/images/notice.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox_dangtb">
     <property name="title">
      <string>Đăng thông báo mới</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Tiêu đề  </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_tieude"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Nội dung</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plainTextEdit_noidung"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_nguoi_tgian">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_ndang">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Người đăng</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_ndang"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_tgian">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Thời gian</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_tgian"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_dang">
          <property name="toolTip">
           <string>Đăng thông báo mới</string>
          </property>
          <property name="statusTip">
           <string>Đăng thông báo mới</string>
          </property>
          <property name="text">
           <string>Đăng</string>
          </property>
          <property name="icon">
           <iconset resource="src.qrc">
            <normaloff>:/images/52-128.png</normaloff>:/images/52-128.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>21</width>
            <height>21</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_capnhat">
          <property name="text">
           <string>OK</string>
          </property>
          <property name="icon">
           <iconset resource="src.qrc">
            <normaloff>:/images/52-128.png</normaloff>:/images/52-128.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>21</width>
            <height>21</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_huy">
          <property name="text">
           <string>Hủy</string>
          </property>
          <property name="icon">
           <iconset resource="src.qrc">
            <normaloff>:/images/button_cancel.png</normaloff>:/images/button_cancel.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>21</width>
            <height>21</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_xemtb">
     <property name="title">
      <string>Tất cả thông báo</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="11,1">
      <item>
       <widget class="QTableView" name="tableView_thongbao"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_refresh">
          <property name="toolTip">
           <string>Cập nhật</string>
          </property>
          <property name="statusTip">
           <string>Cập nhật lại thông báo</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="src.qrc">
            <normaloff>:/images/update.png</normaloff>:/images/update.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_sua">
          <property name="toolTip">
           <string>Sửa thông báo</string>
          </property>
          <property name="statusTip">
           <string>Sửa thông báo</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="src.qrc">
            <normaloff>:/images/Pencil-icon.png</normaloff>:/images/Pencil-icon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>21</width>
            <height>21</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_xoa">
          <property name="toolTip">
           <string>Xóa</string>
          </property>
          <property name="statusTip">
           <string>Xóa thông báo</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="src.qrc">
            <normaloff>:/images/button_cancel.png</normaloff>:/images/button_cancel.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>21</width>
            <height>21</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_xem">
          <property name="toolTip">
           <string>Chi tiết</string>
          </property>
          <property name="whatsThis">
           <string>xem chi tiết thông báo</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="src.qrc">
            <normaloff>:/images/info.png</normaloff>:/images/info.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>21</width>
            <height>21</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
